### PR TITLE
Update color scheme for donut charts 

### PR DESCRIPTION
**Pull request by hqtan**
## Overview of the Pull Request:
this PR updates the colors used in the donut charts (in the 'Trip count and Travel Methods' section of results page) to match the color scheme of the page, and align with the [final figma design](https://www.figma.com/file/DEa98hc9qYB0meam2wtOau/Delta-D?node-id=2921%3A42965) (see screenshot below):

image

## link to Trello card or Github issue
https://trello.com/c/T943YdMY

## How Has This Been Tested?

check in the preview deployment that the color scheme of donut chart matches that of the figma design:
image

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Review by OlaShabalina — APPROVED** [review]
Wow, great job @hqtan !

File: components/ResultsPageComponents/TripCountAndTravelMethods/TripCountAndTravelMethods.jsx [hunk 1]
```diff
@@ -66,11 +67,20 @@ export default function TripCountAndTravelMethods({ dataAboutTrips }) {
                   <PieChartComponent
                     title="Active-public-shared travel"
                     data={activePublicSharedTravel}
+                    colorList={[
+                      "#044B7F",
+                      "#9BB7CC",
+                      "#366F99",
+                      "",
```
Comment (OlaShabalina) [inline components/ResultsPageComponents/TripCountAndTravelMethods/TripCountAndTravelMethods.jsx:74]: actually, what is the empty string for?

Comment (hqtan) [reply components/ResultsPageComponents/TripCountAndTravelMethods/TripCountAndTravelMethods.jsx:74]: haha, good find @OlaShabalina ! 🤣 

It's a lazy hack I did to make the color segments look little more 'balanced', and less dark. If I omitted the empty string, the donut chart looks like this:
image

Do you prefer the chart above? I can update the chart if the one above looks better 👍🏽

Comment (OlaShabalina) [reply components/ResultsPageComponents/TripCountAndTravelMethods/TripCountAndTravelMethods.jsx:74]: Haha, nice catch, agree with you, it's better with the skipped colour! 👌🏻

**Pull request merged**